modules/installing-ocp-agent-basic-inputs.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_with_agent_based_installer/installing-with-agent-basic.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-ocp-agent-basic-inputs_{context}"]
+= Creating the configuration inputs
+
+
+Use this procedure to create the configuration inputs used to create the agent image.
```

Comment:
```suggestion
You create the configuration inputs to create the agent image.
```

- Same comment about self-ref 

Does "configuration inputs used to create" mean -- "create the configuration inputs to create the agent image." ?

Reply:
It means you create configuration files that the installer program will consume when it builds the Agent image, I can try wording it more clearly (basically all of the content in this PR was copied from the existing Agent installer docs)